src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    static final int MAX_NUMBER = 100;
```

Comment:
Static methods and fields are generally considered bad practice in object-oriented programming, as they can lead to issues with inheritance and testability. Consider removing the `static` modifier and making `MAX_NUMBER` an instance variable if it's supposed to change with instances, or a constant if it's truly constant for all instances.